.github/workflows/regression-test-docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: "Regression Test - Docker Image"
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '**regression-test-docker-image.yml'
+      - 'software-full.json'
+      - 'software-sources.json'
+      - 'package.json'
+      - 'update_software.py'
+      - 'SAMPLE-software-rules.json'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the Docker image
+      run: docker build -t "webperf-runner:latest" .
```

Comment:
One last question:
Is it possible to add test against webperf-core here?
similar to:
https://github.com/sitespeedio/sitespeed.io/blob/main/.github/workflows/sitespeed-io-action-example.yml
but preferably in the format of our other test like:
https://github.com/Webperf-se/webperf_core/blob/main/.github/workflows/regression-test-404.yml

So we know webperf_core is actually working.
I don't know what is possible regarding time spent.
best would of course be to test every sinlge test, dont know how long that would take.
but at least maybe start with one core test for like 404 test?
what do you say?